looptribe/form-spatial/0.0/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "container": {
+        "looptribe.formspatial.google_maps_api_key": "%env(LOOPTRIBE_FORMSPATIAL_GOOGLE_MAPS_API_KEY)%"
```

Comment:
Do you really want to put this in the container parameters?
It is good practice to use bundle configuration and then use the bundles extension to move the value to the correct service.